tests/jballerina-unit-test/src/test/resources/test-src/execution/proj3/main.bal

```diff
@@ -1,5 +1,5 @@
-import a;
-import b;
+import unit_tests/proj3.a;
+import unit_tests/proj3.b;
```

Comment:
Don't we have to change the usages of these modules ie. listener a:ABC ep = new a:ABC("ModC");

Reply:
They are already being used as you suggested
https://github.com/ballerina-platform/ballerina-lang/blob/fbead5e74967aa7e4acf2c935e1caef645d1d5f3/tests/jballerina-unit-test/src/test/resources/test-src/execution/proj3/main.bal#L14

Reply:
fixed